core/platform/models.py

```diff
@@ -113,59 +152,127 @@ def import_email_services(cls):
 
     @classmethod
     def import_memcache_services(cls):
+        """Importing the required memcache services
+
+        Returns:
+            Module gae_memcache_services from core.platform.memcache
+        """
         from core.platform.memcache import gae_memcache_services
         return gae_memcache_services
 
     @classmethod
     def import_taskqueue_services(cls):
+        """Importing the required task queue services
+
+        Returns:
+            Module gae_taskqueue_services from core.platform.taskqueue
+        """
         from core.platform.taskqueue import gae_taskqueue_services
         return gae_taskqueue_services
 
     @classmethod
     def import_search_services(cls):
+        """Importing the required search services
+
+        Returns:
+            Module gae_search_services from core.platform.search
+        """
         from core.platform.search import gae_search_services
         return gae_search_services
 
     NAME = 'gae'
 
 
 class Registry(object):
+    """Model through which _Gae_ model's methods are accessed.
+    This is the model accessed by all submodules
+
+    """
+
+    #mapping methods of this model to the methods of _Gae_ model
```

Comment:
This comment is wrong; the values in this dict are not methods. Instead:

`# Maps platform names to the corresponding module registry classes.`